packages/table/src/headers/_headers.scss

```diff
@@ -294,10 +294,8 @@ $dark-selectable-header-menu-selected-hover-background: menu-background(
   &.#{$ns}-editable-text {
     &::before {
       border-radius: 0;
-      bottom: -$cell-padding-vertical;
-      left: -$cell-padding-horizontal + $cell-border-width - 2px;
-      right: -$cell-padding-horizontal + $cell-border-width - 1px;
-      top: -$cell-padding-vertical + $cell-border-width - 2px;
+      inset: -$cell-padding-vertical + $cell-border-width - 2px -$cell-padding-horizontal + $cell-border-width -
+        1px -$cell-padding-vertical -$cell-padding-horizontal + $cell-border-width - 2px;
```

Comment:
Is the value of this `inset` property equivalent to the above?

Also, the formatted result here is a bit harder to read vs. the original. Is there any way we can break this back out into multiple lines?

Reply:
We can keep the original and just disable the rule that prefers a single `inset` property.